test/functionalTest/cases/499_context_providers_basic/update_redirect_delete_operation.test

```diff
@@ -268,34 +263,49 @@ Date: REGEX(.*)
         </contextAttributeList>
       </contextElement>
       <statusCode>
-        <code>200</code>
-        <reasonPhrase>OK</reasonPhrase>
-        <details>Redirected to context provider localhost:REGEX(\d+)/v1</details>
+        <code>404</code>
```

Comment:
Diferent from 472 becoming to 404 (or viceversa), which are not big problem, in this case a 200 has become a 404. Is this ok?